src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheckTest.java

```diff
@@ -153,6 +169,8 @@ public void testTop() throws Exception {
         checkConfig.addAttribute("option", "top");
         final String[] expected = {
             "4: " + getCheckMessage(MSG_ORDERING, "java.awt.Button.ABORT"),
+            "12: " + getCheckMessage(MSG_SEPARATED_IN_GROUP, "java.io.IOException"),
```

Comment:
comment is not a blank line.
We need to update documentation to make it clear that we treat comments as empty lines too.
https://github.com/checkstyle/checkstyle/blob/master/src/xdocs/config_imports.xml#L1230

Reply:
There is `import groups aren't separated internally: ensures that each group aren't separated internally by blank line or comment` in documentation.
Should I update it?

Reply:
we need to update https://github.com/checkstyle/checkstyle/blob/master/src/xdocs/config_imports.xml#L1292

Reply:
Done.